sjb/config/test_cases/azure_build_node_image_centos_311.yml

```diff
@@ -0,0 +1,112 @@
+---
+provision:
+  os: "rhel"
+  stage: "base"
+  provider: "aws"
+sync:
+  - openshift,openshift-ansible=release-3.11
```

Comment:
There is no such branch (yet). Use master.

Reply:
tempted to delete all these for now. I was not expting to get config for jenkis so fast :D